helm-util/helm.libsonnet

```diff
@@ -0,0 +1,57 @@
+local d = import 'github.com/sh0rez/docsonnet/doc-util';
+{
+  local this = self,
+
+  '#': d.pkg(
```

Comment:
Docsonnet keys should be private (`::`), because they never need to be evaluated out of docsonnet

Reply:
Just mimicking the docs as written in k8s-alpha ;)

Reply:
In k8s-alpha, these are already created as private by the generator. `+:` maintains previous visibility, so this is not a problem there.

Here you create those keys the first time, it inherits public visibility, which is an issue when importing the library like so:

```
(import "helm-util/helm.libsonnet") {
}
```

Because now downstream code needs access to `doc-util` and also the docsonnet keys will be in the output